### PR TITLE
[Route] Get Genre By Id

### PR DESCRIPTION
**Pull request by Ahuanmarca**

### CODE REVIEW

File: src/api/genres/genres.router.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import Router from 'express';
+import * as genresController from './genres.controller.js';
+
+const router = Router();
+
+router.get('/byId/:id', genresController.getGenreById);
```
Comment (nuinn) [inline src/api/genres/genres.router.js]: The uniform byId router for the project has been '/:id/' without '/byId/', '/id/', etc. Would you like to keep this router?

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,4 @@
 node_modules
 .env
 coverage
-
+.prettierrc
```
Comment (andreamir) [inline .gitignore:4]: For those who do not have Prettier installed, this line is not necessary.
However, it doesn't hurt since it can prevent conflicts. 🥰

**Review by jaanmeld — APPROVED** [review]

**Review by nuinn — CHANGES_REQUESTED** [review]
In terms of functionality, it's great. But I don't know if you want to change the router path to match the others in the project before we merge it?

**Review by andreamir — COMMENTED** [review]
awaiting the route byId/:id to be changed

**Review by nuinn — APPROVED** [review]

**Review by andreamir — APPROVED** [review]

File: src/api/router.js [hunk 3]
```diff
@@ -1,13 +1,15 @@
...
+// import authRouter from './auth/auth.router.js';
 import gameTitlesRouter from './gameTitles/gameTitles.router.js';
 import platformsRouter from './platforms/platforms.router.js';
 import productsRouter from './products/products.router.js';
 
 const router = Router();
 
-router.use('/auth', authRouter);
+router.use('/genres', genresRouter);
+// router.use('/auth', authRouter);
```
Comment (JonaTheBridge) [inline src/api/router.js]: Todas estas lineas comentadas, si se comentan porque dan error, prefiero que las elimines y que las cree el que las necesite.

**Review by JonaTheBridge — CHANGES_REQUESTED** [review]

**Review by JonaTheBridge — APPROVED** [review]

**Pull request merged**